### PR TITLE
feat: Add view `wholesale_edi_results.energy_result_points_per_ga_v1`

### PR DESCRIPTION
**Pull request by BjarkeMeier**
# Description

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__succeeded_energy_result_v1.sql
Comment (JesperGomes) [inline source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__succeeded_energy_result_v1.sql:1]: I assume it is on purpose with the double underscores?

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__succeeded_energy_result_v1.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+           WHEN e.time_series_type = 'positive_grid_loss' THEN 'consumption'
+           WHEN e.time_series_type = 'temp_flex_consumption' THEN 'consumption'
+           WHEN e.time_series_type = 'temp_production' THEN 'production'
+       END as metering_point_type,
+       CASE
+           WHEN e.time_series_type = 'non_profiled_consumption' THEN 'non_profiled'
+           WHEN e.time_series_type = 'flex_consumption' THEN 'flex'
+           WHEN e.time_series_type = 'temp_flex_consumption' THEN 'flex'
+       END as settlement_method,
+    e.metering_point_id
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__succeeded_energy_result_v1.sql:41]: wrong indent?

File: source/databricks/calculation_engine/tests/features/utils/dataframes/edi_results/energy_result_points_per_ga_v1_view_schema.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        t.StructField("calculation_period_start", t.TimestampType(), False),
+        t.StructField("calculation_period_end", t.TimestampType(), False),
+        t.StructField("calculation_version", t.LongType(), False),
+        t.StructField("result_id", t.StringType(), False),
+        t.StructField("grid_area_code", t.StringType(), False),
+        t.StructField("metering_point_type", t.StringType(), False),
+        t.StructField("settlement_method", t.StringType(), True),
+        t.StructField(
+            "resolution", t.StringType(), True
+        ),  # TODO BJM: Should this be False?
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/tests/features/utils/dataframes/edi_results/energy_result_points_per_ga_v1_view_schema.py:30]: Yes

File: source/databricks/calculation_engine/tests/features/utils/dataframes/edi_results/energy_result_points_per_ga_v1_view_schema.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        t.StructField("time", t.TimestampType(), False),
+        t.StructField("quantity", t.DecimalType(18, 3), False),
+        t.StructField("unit", t.StringType(), False),
+        t.StructField(
+            "quantity_qualities", t.ArrayType(t.StringType(), True), False
+        ),  # TODO BJM: Should this be False?
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/tests/features/utils/dataframes/edi_results/energy_result_points_per_ga_v1_view_schema.py:36]: Yes

File: source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__succeeded_energy_result_v1.sql
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/package/datamigration/current_state_scripts/views/create__succeeded_energy_result_v1.sql:1]: Yes. Sorting in order to ensure that the dependent view is executed before the dependee.

File: source/databricks/calculation_engine/tests/features/utils/views/assertion.py [hunk 4]
```diff
@@ -28,12 +29,19 @@ def assert_output(
     actual_result = _get_expected_for_output(actual_results, output_name)
     expected_result = _get_expected_for_output(expected_results, output_name)
 
+    columns_to_skip = []
+    if ResultColumnNames.calculation_result_id in expected_result.columns:
+        columns_to_skip.append(ResultColumnNames.calculation_result_id)
+    if "result_id" in expected_result.columns:
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/tests/features/utils/views/assertion.py:35]: When do we use this functionality vs IGNORED?

File: source/databricks/calculation_engine/tests/features/utils/dataframes/edi_results/energy_result_points_per_ga_v1_view_schema.py (hunk 2 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/tests/features/utils/dataframes/edi_results/energy_result_points_per_ga_v1_view_schema.py:30]: But do we bother to solve this? I think we should rather ignore nullability...

File: source/databricks/calculation_engine/tests/features/utils/views/assertion.py (hunk 4 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/tests/features/utils/views/assertion.py:35]: I don't understand the question.

File: source/databricks/calculation_engine/tests/features/utils/dataframes/edi_results/energy_result_points_per_ga_v1_view_schema.py (hunk 2 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/tests/features/utils/dataframes/edi_results/energy_result_points_per_ga_v1_view_schema.py:30]: We'll get back to this.

File: source/databricks/calculation_engine/tests/features/utils/dataframes/edi_results/energy_result_points_per_ga_v1_view_schema.py (hunk 3 above)
Comment (BjarkeMeier) [reply source/databricks/calculation_engine/tests/features/utils/dataframes/edi_results/energy_result_points_per_ga_v1_view_schema.py:36]: We'll get back to this.

**Pull request merged**